scrapy_zyte_api/handler.py

```diff
@@ -95,9 +93,12 @@ async def _download_request(
         api_data = {**{"url": request.url}, **api_params}
         if self._job_id is not None:
             api_data["jobId"] = self._job_id
+        retrying = request.meta.get("zyte_api_retry_policy") or self._retry_policy
         try:
             api_response = await self._client.request_raw(
-                api_data, session=self._session
+                api_data,
+                session=self._session,
+                retrying=retrying,
```

Comment:
minor: I was thinking if we should set `handle_retries=True` to ensure that `retrying` is always used in the client: https://github.com/zytedata/python-zyte-api/blob/main/zyte_api/aio/client.py#L111-L112

Reply:
Given that is the default value, and this is a public API (i.e. the default value should stay like that for backward compatibility), I don’t think it is necessary, but I do not mind adding it either (explicit better than implicit?).

Reply:
Oh, I forgot we allowed custom clients. I actually see your point now, if the configured retry policy is anything other than `None` maybe we should pass `handle_retries=True` as well.

That said, custom clients with `handle_retries=False` as default could also be smart about `retrying`, i.e. if `retrying` is not `None`, ignore `handle_retries` and work as if `handle_retries=True`.

All-in-all, I think the best solution going forward may be to deprecate `handle_retries` and instead provide a `NO_RETRIES` retry policy that can be passed to `retrying` to disable retries. And maybe rename `retrying` to `retry_policy`.

I will drop the idea as suggested, but I am completely OK with forcing `handle_retries=True` if a custom retrying policy is defined in settings or request metadata, so let me know if you change your mind.